packages/cli/src/commands/governance/list.ts

```diff
@@ -0,0 +1,33 @@
+import { concurrentMap } from '@celo/utils/src/async'
+import { zip } from '@celo/utils/src/collections'
+import chalk from 'chalk'
+import { BaseCommand } from '../../base'
+import { printValueMap } from '../../utils/cli'
+
+export default class List extends BaseCommand {
+  static description = 'List live governance proposals (queued and ongoing)'
+
+  static flags = {
+    ...BaseCommand.flags,
+  }
+
+  static examples = ['list']
+
+  async run() {
+    this.parse(List)
+
+    const governance = await this.kit.contracts.getGovernance()
+    const queue = await governance.getQueue()
+    const sortedQueue = governance.sortedQueue(queue)
+
+    console.log(chalk`{purple.bold Queued Proposals:}`)
+    sortedQueue.map(printValueMap)
+
+    const dequeue = await governance.getDequeue()
+    const stages = await concurrentMap(5, dequeue, (id) => governance.getProposalStage(id))
```

Comment:
Optional nit: You can just pass through `governance.getProposalStage` without the lambda? (unless there is more than one parameter)